app/components/shared/rejection_reasons/rejection_reasons_component.html.erb

```diff
@@ -25,6 +25,7 @@
         <% else %>
           <p><%= I18n.t("rejection_reasons.#{reason.id}.description") %></p>
         <% end %>
+        <%= link_to_find_when_rejected_on_qualifications(application_choice) if @render_link_to_find_when_rejected_on_qualifications %>
```

Comment:
should this link only be shown on qualification questions and not on each rejection reason?